source/simulator/parameters.cc

```diff
@@ -87,6 +87,18 @@ namespace aspect
                        "steps. This does not include the last refinement step before moving to timestep 1. "
                        "When this parameter has a larger value than max nonlinear iterations, the latter is used.");
 
+    prm.declare_entry ("Max pre-Newton nonlinear iterations", "10",
```

Comment:
Or at least start a subsection "Solver parameters" ... all those solver related parameters in the top-level are annoying already.